src/VisualStudio/Core/SolutionExplorerShim/AnalyzersCommandHandler.cs

```diff
@@ -159,7 +154,7 @@ private void UpdateMenuItemVisibility()
                                                 Path.GetExtension(itemName).Equals(".ruleset", StringComparison.OrdinalIgnoreCase);
         }
 
-        private void UpdateMenuItemsChecked()
+        private void UpdateSeverityMenuItemsChecked()
         {
             _setSeverityErrorMenuItem.Checked = AnyDiagnosticsWithSeverity(ReportDiagnostic.Error);
             _setSeverityWarningMenuItem.Checked = AnyDiagnosticsWithSeverity(ReportDiagnostic.Warn);
```

Comment:
No, this is the desired behavior.

Now, when I add "Default" as an option (part of the work for #468) I may also change what the check mark means (should it show the *effective* severity, or the *rule set* severity, since the rule set is what you're actually changing?) and may revisit this choice as well.